src/js/services/involvement-API/getLikes.js

```diff
@@ -0,0 +1,17 @@
+import involvementAPI from './involvement-API.js';
+
+// Function to get the likes of each Show  -----------------------------------------------------
+// eslint-disable-next-line consistent-return
+const getLikesData = async () => {
+  const response = await fetch(`${involvementAPI.baseUrl}${involvementAPI.appId}/likes/`, { method: 'GET' });
+  if (JSON.stringify(response) !== '[]') {
+    const data = await response.json();
+    return data;
+  }
+  if (JSON.stringify(response) === '[]') {
+    const data = [];
+    return data;
+  }
+};
```

Comment:
Hi @Ruthmy partner! 😄, It would be a good idea and best practice to not repeat yourself, in this case, you can abstract these two conditionals into one! 🌟 

e.g

```js
const data = await response.json();
return JSON.stringify(data) === '[]' ? [] : data;
```
This approach eliminates the need to check the length of the string representation and simplifies the code further.

What do you think? 😃